src/common/image.c

```diff
@@ -113,7 +113,7 @@ int dt_image_is_raw(const dt_image_t *img)
   const char *dt_non_raw_extensions[]
       = { ".jpeg", ".jpg",  ".pfm", ".hdr", ".exr", ".pxn", ".tif", ".tiff", ".png",
           ".j2c",  ".j2k",  ".jp2", ".jpc", ".gif", ".jpc", ".jp2", ".bmp",  ".dcm",
-          ".jng",  ".miff", ".mng", ".pbm", ".pnm", ".ppm", ".pgm", NULL };
+          ".jng",  ".miff", ".mng", ".pbm", ".pnm", ".ppm", ".pgm", ".dng", NULL };
 
   if(img->flags & DT_IMAGE_RAW) return TRUE;
```

Comment:
I would add comment to make this clear:

// rawspeed set the DT_IMAGE_RAW to DNG file which are RAW. The non RAW DNG file will be taken into
// account below.

Or something like that.

Reply:
done with second commit.